src/main/resources/avro/common.avdl

```diff
@@ -2,6 +2,18 @@
 /**
 This protocol defines common types used in the other GA4GH protocols. It does
 not have any methods; it is merely a library of types.
+
+Many of these types are related to definign sequence graphs. A sequence graph is
```

Comment:
Alright, I've fixed that in my branch. Thanks!

On Thu, Jan 22, 2015 at 12:13 PM, Ryan Williams <notifications@github.com>
wrote:

> In src/main/resources/avro/common.avdl
> <https://github.com/ga4gh/schemas/pull/198#discussion_r23405870>:
>
> > @@ -2,6 +2,18 @@
> >  /**
> >  This protocol defines common types used in the other GA4GH protocols. It does
> >  not have any methods; it is merely a library of types.
> > +
> > +Many of these types are related to definign sequence graphs. A sequence graph is
>
> *defining :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga4gh/schemas/pull/198/files#r23405870>.
>